crates/nu-command/tests/commands/which.rs

```diff
@@ -105,3 +105,16 @@ fn do_not_show_hidden_commands() {
     let length: i32 = actual.out.parse().unwrap();
     assert_eq!(length, 0);
 }
+
+#[test]
+fn which_accepts_splatted_list() {
```

Comment:
Nit: for consistency as we refer to it as the spread operator.
```suggestion
fn which_accepts_spread_list() {
```